packages/eventra-events/.env.example

```
@@ -0,0 +1,8 @@
+﻿DATABASE_URL=<REPLACE_ME>
```

dotenv-linter (3.3.0)

[warning] [LeadingCharacter] Invalid leading character detected

(LeadingCharacter)